wandb/integration/keras/callbacks/model_checkpoint.py

```diff
@@ -0,0 +1,77 @@
+import os
+import wandb
+import shutil
+
+from tensorflow.keras import callbacks
+
+from ..keras import patch_tf_keras
+
+
+patch_tf_keras()
+
+
+class WandbModelCheckpointCallback(callbacks.Callback):
```

Comment:
* I ran this callback with default setting. I noticed that the model is logged as W&B file in the `.h5` format. [run page](https://wandb.ai/ayut/mirbind/runs/rb70gkyn/files?workspace=user-ayut)
* When I used `save_weights_only=True` there wasn't anything logged even as W&B file. [run page](https://wandb.ai/ayut/mirbind/runs/3vsl41ti/files?workspace=)
* Using `save_model_as_artifact=True` worked as expected.
* Using both `save_weights_only=True` and `save_model_as_artifact=True`  gave this error: `ValueError: Path is not a directory: /home/ayusht/wandb-cli/client/wandb/run-20220817_080253-35ry19kd/files/model-best`.

Reply:
On it...